Makefile

```diff
@@ -176,7 +176,7 @@ ZKEVM_MODULES_FOR_REFERENCE_TESTS := ${ALU} \
 				     ${EUC} \
 				     ${EXP} \
 				     ${GAS} \
-				     ${HUB_COLUMNS} \
+				     ${HUB} \
```

Comment:
This is in `ZKEVM_MODULES_FOR_REFERENCE_TESTS`, which in any case blows up all over the place.